rocketpy/prints/flight_prints.py

```diff
@@ -369,6 +369,28 @@ def maximum_values(self):
                 self.flight.max_acceleration_time,
             )
         )
+        # check if max acceleration during burn is different than overall
+        # max acceleration
+        if (
+            self.flight.max_acceleration_time
+            != self.flight.max_acceleration_power_on_time
+        ):
+            print(
+                "Maximum Acceleration During Motor Burn: {:.3f} m/s² at {:.2f} s".format(
+                    self.flight.max_acceleration_power_on,
+                    self.flight.max_acceleration_power_on_time,
+                )
+            )
```

Comment:
Why is the maximum acceleration on motor burn only logged when it differs from the overall maximum acceleration? In my opinion, this is quite an useful information that may always be shown.